aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/services/cognitoidentityprovider/model/transform/StartUserImportJobRequestMarshaller.java

```diff
@@ -22,6 +22,8 @@
 import java.io.StringWriter;
 import java.io.Writer;
 
+import android.text.TextUtils;
```

Comment:
This seems not great that they added an import to TextUtils in all these Marshallers with no apparent use but not a blocker since this is code generated.